src/coreclr/nativeaot/Runtime/windows/PalRedhawkInline.h

```diff
@@ -42,6 +42,13 @@ FORCEINLINE int32_t PalInterlockedExchange(_Inout_ int32_t volatile *pDst, int32
     return _InterlockedExchange((long volatile *)pDst, iValue);
 }
 
+EXTERN_C int64_t _InterlockedExchange64(int64_t volatile *, int64_t);
+#pragma intrinsic(_InterlockedExchange64)
```

Comment:
This line throws an error, I guess there should be other way do a 64 bit InterlockedExchange

Reply:
It would be best to just depend on Windows SDK for these intrinsics (ie avoid the method with underscore and the #pragma):
```
...
FORCEINLINE int32_t PalInterlockedExchange(_Inout_ int32_t volatile *pDst, int32_t iValue)
{
    return InterlockedExchange((long volatile *)pDst, iValue);
}

FORCEINLINE int64_t PalInterlockedExchange64(_Inout_ int64_t volatile *pDst, int64_t iValue)
{
    return InterlockedExchange64(pDst, iValue);
}
....
```

Feel free to do it for all methods in this file.

(You may need to `#include <windows.h>` somewhere to make this work.)